diffkemp/simpll/Utils.h

```diff
@@ -129,4 +129,15 @@ std::string getIdentifierForValue(const Value *Val,
 /// methods.
 std::string valueToString(const Value *Val);
 
+/// Get a string matching the current indentation level.
+/// \param offset Indentation level offset to use, defaults to zero.
+/// \param prefixChar Indentation prefix character, defaults to space.
+std::string getDebugIndent(const int offset = 0, const char prefixChar = ' ');
+
+/// Increase the level of debug indentation by one.
+void increaseIndentLevel();
```

Comment:
Use rather `increaseDebugIndentLevel` or `increaseDebugIndent` to keep it shorter. It should be clear that these functions deal with debugging messages indent.

Reply:
Here I will stick to the longer version, even though I skipped the `Debug` part in the function name to make it shorter. I believe it should be clearer that this function doesn't change the number of indentation characters, but the current indentation nesting level (currently two spaces per level by default).

Reply:
This function name length is actually not that bad considering there are longer function names like `getSubstringToMatchingBracket` or `convertInlineAsmToLLVMFormat` in SimpLL.